src/main/java/net/dries007/tfc/util/fuel/FuelManager.java

```diff
@@ -41,34 +38,52 @@ public static boolean isItemForgeFuel(ItemStack stack)
         return fuel != EMPTY && fuel.isForgeFuel();
     }
 
+    public static boolean isItemBloomeryFuel(ItemStack stack)
+    {
+        Fuel fuel = getFuel(stack);
+        return fuel != EMPTY && fuel.isBloomeryFuel();
+    }
+
     public static void postInit()
     {
         for (Tree wood : TFCRegistries.TREES.getValuesCollection())
         {
             BlockLogTFC log = BlockLogTFC.get(wood);
-            FUELS.add(new Fuel(new ItemStack(log), wood.getBurnTicks(), wood.getBurnTemp()));
+            FUELS.add(new Fuel(IIngredient.of(new ItemStack(log)), wood.getBurnTicks(), wood.getBurnTemp()));
         }
 
-        // Coal (Vanilla)
-        FUELS.add(new Fuel(new ItemStack(Items.COAL, 1, 0), 2200, 1350f, true));
-        // Coal (TFC Variants)
-        FUELS.add(new Fuel(ItemOreTFC.get(TFCRegistries.ORES.getValue(DefaultMetals.BITUMINOUS_COAL), 1), 2200, 1415f, true));
-        FUELS.add(new Fuel(ItemOreTFC.get(TFCRegistries.ORES.getValue(DefaultMetals.LIGNITE), 1), 2200, 1415f, true));
+        // Coals
+        FUELS.add(new Fuel(IIngredient.of("gemCoal"), 2200, 1415f, true, true));
+        FUELS.add(new Fuel(IIngredient.of("gemLignite"), 2000, 1350f, true, true));
```

Comment:
Instead of using ore dictionary values, use the item instance here to differentiate. (Since they should both really be `gemCoal` for purposes of other mods)